trestle/tasks/osco_to_oscal.py

```diff
@@ -0,0 +1,184 @@
+
+    def simulate(self) -> TaskOutcome:
+        """Provide a simulated outcome."""
+        try:
+            if self._config:
+                idir = pathlib.Path(self._config.get('input-dir'))
+                if idir is None:
+                    logger.error(f'config missing "input-dir"')
+                    return TaskOutcome('simulated-failure')
+                odir = pathlib.Path(self._config.get('output-dir'))
+                if odir is None:
+                    logger.error(f'config missing "output-dir"')
+                    return TaskOutcome('simulated-failure')
+                overwrite = self._config.getboolean('output-overwrite', True)
+                for ifile in sorted(pathlib.Path(idir).iterdir()):
+                    parts = ifile.parts
+                    ifn = parts[len(parts)-1]
+                    if ifn.endswith('oscal-metadata.yaml'):
+                        continue
+                    if ifn.endswith('oscal-metadata.yml'):
+                        continue
+                    ofile = self._calculate_ofile(ifn, odir)
+                    if not overwrite:
+                        if os.path.exists(ofile):
+                            logger.error(f'file exists: {ofile}')
+                            return TaskOutcome('simulated-failure')
+                    mfile = idir / 'oscal-metadata.yaml'
+                    metadata = self._get_metadata(mfile)
+                    logger.debug(f'create: {ofile}')
+                    idata = self._read_content(ifile)
+                    odata, analysis = osco.get_observations(idata, metadata)
+                    self._write_content(ofile, odata, True)
+                    logger.debug(f'Rules Analysis:')
+                    logger.debug(f'config_maps: {analysis["config_maps"]}')
+                    logger.debug(f'dispatched rules: {analysis["dispatched_rules"]}')
+                    logger.debug(f'result types: {analysis["result_types"]}')
+                return TaskOutcome('simulated-success')
+            logger.error(f'config missing')
+            return TaskOutcome('simulated-failure')
+        except Exception:
+            traceback.print_exc()
+            return TaskOutcome('simulated-failure')
+
+    def execute(self) -> TaskOutcome:
+        """Provide an actual outcome."""
+        try:
+            if self._config:
+                idir = pathlib.Path(self._config.get('input-dir'))
+                if idir is None:
+                    logger.error(f'config missing "input-dir"')
+                    return TaskOutcome('failure')
+                odir = pathlib.Path(self._config.get('output-dir'))
+                if odir is None:
+                    logger.error(f'config missing "output-dir"')
+                    return TaskOutcome('failure')
+                overwrite = self._config.getboolean('output-overwrite', True)
+                quiet = self._config.getboolean('quiet', False)
```

Comment:
Trestle task should observe the verbosity flag. What that implies is if a user passes -V to the command line it will set the log level to print out debug. I think it would make more sense to align with this paradigm.